src/common/Index.js

```diff
@@ -0,0 +1,269 @@
+import React from "react";
+import Helmet from "react-helmet";
+import stringSimilarity from "string-similarity";
+import { unescape, shuffle } from "lodash";
+import { format, parse as parseDate } from "date-fns";
+import Slider from "infinite-react-carousel";
+import { Post } from "../templates/post";
+import { TitleWithAnchor } from "./TitleWithAnchor";
+
+export const BlogLatest = ({title, posts}) => (
```

Comment:
@qmonnet you've done a great job on DRYing the code actually 👍

As part of a further improvement, I would only suggest splitting the file into smaller chunks. `Index.js` is over 200 LOC and tend to grow up in the future. So, a separate file per each component can contribute into the maintainability. Something like this:
* `src/common/homepage/BlogLatest.js`
* `src/common/homepage/BlogRoll.js`
* `src/common/homepage/Helmet.js`
* `src/common/homepage/MainTitle.js`
* `src/common/homepage/Section.js`
* `src/common/homepage/Videos.js`

Reply:
Yep, I considered it but was unsure. I can certainly do that before we merge. Thanks!